### PR TITLE
replace pkg/errors and bump related library

### PR DESCRIPTION
**Pull request by zouyee**
replace pkg/errors and bump related(console、container and go-cni etc.) library

Signed-off-by: Zou Nengren <zouyee1989@gmail.com>

### CODE REVIEW

**Comment by stefanberger** [comment]
Are other projects replacing this package as well these days?

**Comment by zouyee** [comment]
> Are other projects replacing this package as well these days?

Yes

File: cmd/ctr/commands/images/pull.go [hunk 1]
```diff
@@ -110,13 +110,13 @@ command. As part of this process, we do the following:
...
 			if err != nil {
-				return errors.Wrap(err, "unable to resolve image platforms")
+				return fmt.Errorf("unable to resolve image platforms: %w", err)
 			}
 		} else {
 			for _, s := range context.StringSlice("platform") {
 				ps, err := platforms.Parse(s)
 				if err != nil {
-					return errors.Wrapf(err, "unable to parse platform %s", s)
+					return fmt.Errorf("unable to parse platform %s; %w", s, err)
```
Comment (lumjjb) [inline cmd/ctr/commands/images/pull.go:119]:
```suggestion
					return fmt.Errorf("unable to parse platform %s: %w", s, err)
```

**Review by lumjjb — APPROVED** [review]
Awesome - thanks so much for the PR @zouyee ! 1 very minor nit

File: cmd/ctr/commands/images/pull.go (hunk 1 above)
Comment (zouyee) [reply cmd/ctr/commands/images/pull.go]: thanks, done

**Comment by stefanberger** [comment]
LGTM. Thanks.

**Review by lumjjb — APPROVED** [review]
LGTM!

**Pull request merged**